README.md

```diff
@@ -49,3 +49,29 @@ The special ``ProphecyTestCase`` exposes a method ``prophesize($classOrInterface
 to use Prophecy.
 For the usage of the Prophecy doubles, please refer to the [Prophecy documentation](https://github.com/phpspec/prophecy).
 
+If you need to the Prophecy integration alongside a custom base TestCase rather than the PHPUnit one, a trait is available with all the necessary logic, except the override of the PHPUnit `verifyMockObjects` method (which cannot be provided by a trait). Use it like that:
```

Comment:
```suggestion
If you need the Prophecy integration alongside a custom base TestCase rather than the PHPUnit one, a trait is available with all the necessary logic, except the override of the PHPUnit `verifyMockObjects` method (which cannot be provided by a trait). Use it like that:
```

Reply:
Actually, I meant `If you need to use the Prophecy integration`